Libplanet/Store/BaseStore.cs

```diff
@@ -138,6 +138,22 @@ public Block<T> GetBlock<T>(BlockHash blockHash)
         /// <inheritdoc cref="IStore.GetTxExecution(BlockHash, TxId)"/>
         public abstract TxExecution GetTxExecution(BlockHash blockHash, TxId txid);
 
+        /// <inheritdoc cref="IStore.PutTxIdBlockHashIndex(Guid, TxId, BlockHash)"/>
+        public abstract void PutTxIdBlockHashIndex(Guid chainId, TxId txId, BlockHash blockHash);
+
+        /// <inheritdoc cref="IStore.HasTxIdBlockHashIndex(Guid, TxId)"/>
+        public bool HasTxIdBlockHashIndex(Guid chainId, TxId txId)
+        {
+            return GetTxIdBlockHashIndex(chainId, txId) != null;
```

Comment:
Although it's a trivial stylistic choice, in modern C# it tends to be like:

```suggestion
            return GetTxIdBlockHashIndex(chainId, txId) is { };
```